cmd/kubelet/app/server.go

```diff
@@ -199,12 +202,14 @@ func NewKubeletServer() *KubeletServer {
 		RootDirectory:     defaultRootDir,
 		SyncFrequency:     10 * time.Second,
 		SystemContainer:   "",
+		ReconcileCIDR:     true,
 	}
 }
 
 // AddFlags adds flags for a specific KubeletServer to the specified FlagSet
 func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.Config, "config", s.Config, "Path to the config file or directory of files")
+	fs.BoolVar(&s.Accepting, "accepting", s.Accepting, "Accepting condition")
```

Comment:
Put that in as a placeholder and forgot about it. I'll add a more descriptive description.